formbuilder/components/EditableField.js

```diff
@@ -62,24 +102,19 @@ export default class EditableField extends Component {
   }
 
   handleDrop(data) {
-    const {name} = this.props;
+    const {name, swapFields, insertField} = this.props;
     if ("moved-field" in data && data["moved-field"]) {
       if (data["moved-field"] !== name) {
-        this.props.swapFields(data["moved-field"], name);
+        swapFields(data["moved-field"], name);
       }
     } else if ("field" in data && data.field) {
-      this.props.insertField(JSON.parse(data.field), name);
+      insertField(JSON.parse(data.field), name);
```

Comment:
Because draggable elements data are attached to a data attribute to the DOM element, and these can only carry strings.